### ISSUE TITLE
CI: Introduce tests for JS-peer

### ISSUE THREAD

**Issue by maschad** [libp2p/universal-connectivity#149]
It would be good to have a basic ping test akin to the one in [js-libp2p ping module](https://github.com/libp2p/js-libp2p/blob/main/packages/protocol-ping/test/index.spec.ts#L45-L59) to ensure that breaking changes are caught at runtime.

**Comment by 2color** [comment]
Do ping module in js-libp2p run in the browser?

**Comment by maschad** [comment]
Yup, we use [aegir](https://github.com/ipfs/aegir) to [run them](https://github.com/libp2p/js-libp2p/blob/main/package.json#L19-L22) 

Aegir uses https://github.com/microsoft/playwright under the hood to run the browser tests.

I've also had a good experience using [vitest's browser mode](https://vitest.dev/guide/browser.html) as well so that would be my recommendation.
